meinberlin/assets/scss/utility.scss

```diff
@@ -193,3 +201,7 @@
         display: none !important;
     }
 }
+
+.u-font-italic {
+    font-style: italic;
+}
```

Comment:
when adding utilities, we try to only add them if it is garenteed we will reuse them so as not to have a huge long list, as 2 of these are for positioning an icon, I would be more inclined to make an icon position class to cover the position with one class name and as it's not a spacer size we are likley to use for any other purpose.

Reply:
Where would you put that? Do we have something for that or is that also a utility?

Reply:
i think for this case it will be re-used in future so can be there as utility class right @phillimorland ?

Reply:
yeah, margin in just one style, if we have to use more then 2 or 3 then it's probably not a utility